src/main/java/internet/store/controllers/InjectDataController.java

```diff
@@ -25,18 +27,11 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        User daniel = new User("Daniel", "dan", "123");
-        userService.create(daniel);
-        shoppingCartService.create(new ShoppingCart(daniel.getUserId()));
-        User sergey = new User("Sergey", "serg", "4321");
-        userService.create(sergey);
-        shoppingCartService.create(new ShoppingCart(sergey.getUserId()));
-        User mandy = new User("Mandy", "Mandy", "1234321");
-        userService.create(mandy);
-        shoppingCartService.create(new ShoppingCart(mandy.getUserId()));
-        User joanne = new User("Joanne", "Joe", "1234321");
-        userService.create(joanne);
-        shoppingCartService.create(new ShoppingCart(joanne.getUserId()));
+        User admin = new User("Admin", "admin", "1");
+        admin.setRoles(Set.of(Role.of("ADMIN")));
+        userService.create(admin);
+        shoppingCartService.create(new ShoppingCart(admin.getUserId()));
```

Comment:
Our home task says that only USERs can access shopping cart operations, so maybe there's no need to create a shopping cart for the ADMIN?

Reply:
make sense